### ISSUE TITLE
Question about PPO Training (Qwen2.5)

### ISSUE THREAD

**Issue by gray311** [OpenRLHF/OpenRLHF#756]
I am using 8 A100 40GB GPUs. The following script for PPO training works successfully. However, when I switch the actor pretrained model to "qwen2.5-7B-instruct", it runs out of VRAM (GPU memory) during the "make experience" phase when calculating reference logits. What could be causing this issue?

```
set -x 

ray job submit --address="http://127.0.0.1:8265" \
   -- python3 -m openrlhf.cli.train_ppo_ray \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 2 \
   --reward_num_nodes 1 \
   --reward_num_gpus_per_node 2 \
   --critic_num_nodes 1 \
   --critic_num_gpus_per_node 2 \
   --actor_num_nodes 1 \
   --actor_num_gpus_per_node 2 \
   --vllm_num_engines 2 \
   --vllm_tensor_parallel_size 2 \
   --colocate_critic_reward \
   --colocate_actor_ref \
   --pretrain OpenRLHF/Llama-3-8b-sft-mixture \ #  switching the actor to Qwen/Qwen2.5-Coder-3B-Instruct cause OOD
   --reward_pretrain Qwen/Qwen2.5-Coder-3B-Instruct \
   --save_path /openrlhf/examples/checkpoint/llama3-8b-rlhf \
   --micro_train_batch_size 16 \
   --train_batch_size 128 \
   --micro_rollout_batch_size 32 \
   --rollout_batch_size 1024 \
   --max_samples 100000 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --generate_max_len 1024 \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 5e-7 \
   --critic_learning_rate 9e-6 \
   --init_kl_coef 0.01 \
   --prompt_data OpenRLHF/prompt-collection-v0.1 \
   --input_key context_messages \
   --apply_chat_template \
   --normalize_reward \
   --packing_samples \
   --adam_offload \
   --flash_attn \
   --ref_reward_offload \
   --gradient_checkpointing \
   --load_checkpoint \
   --use_wandb {wandb_token}
```

**Comment by YangXiaojie1998** [comment]
OOM during training process, I used 4 A6000 Ada (48G) but still have no solutions.
